api/Makefile

```diff
@@ -57,17 +57,17 @@ gen-files .generate_files: lint-cache-dir clean-generated
 	   sh -c '$(GIT_CONFIG_SSH) defaulter-gen \
 		--v 1 --logtostderr \
 		--go-header-file "/go/src/$(PACKAGE_NAME)/hack/boilerplate/boilerplate.go.txt" \
-		--input-dirs "$(PACKAGE_NAME)/pkg/apis/projectcalico/v3" \
```

Comment:
`--input-dirs` flag is removed in https://github.com/kubernetes/code-generator/commit/4d6b3bd5ca9ac09408e6e37d251d14ef46c76d0e.